lib/runner.js

```diff
@@ -146,7 +146,9 @@ function runner(script, payload, options) {
   //
   let runnerPlugins = [];
   _.each(runnableScript.config.plugins, function tryToLoadPlugin(pluginConfig, pluginName) {
-    let requireString = 'artillery-plugin-' + pluginName;
+    let pluginPackage = pluginConfig.package || runnableScript.config.pluginsPackage;
+    let pluginPrefix = pluginPackage ? pluginPackage + '/' : 'artillery-plugin-';
```

Comment:
Yup, it should the full package name (scope + package), not just the scope to which a package name gets appended as is currently implemented in this PR. So whatever is set in the `package` attribute is what gets `require`d.